StepperEngine/src/DataTypes/EnumeratorType.java

```diff
@@ -0,0 +1,18 @@
+package DataTypes;
+
+public class EnumeratorType extends DataType<String> implements UserFriendly {
+
+    public EnumeratorType(String name, boolean isInput) {
+        super(name, name.toLowerCase().replace('_',' '), Type.ENUM, isInput);
+    }
+
+    public EnumeratorType(String enumString, String name, boolean isInput){super(name, name.toLowerCase().replace('_',' '), enumString, Type.ENUM, isInput);}
+
+    @Override
+    public String getPresentableString() {
+        return getData();
+    }
+
+    @Override
+    public void setData(String dataStr) {super.setData(dataStr);}
```

Comment:
Shouldn't we have validation here? To make sure that 'dataStr' a valid value of the enum?

He says in the instructions that this DataType should only be able to hold/accept a valid value.